### ISSUE TITLE
runtime error: invalid memory address or nil pointer dereference

### ISSUE THREAD

**Issue by demndevel** [sheepla/srss#33]
Hi. I ran the `srss u` as usual and after it i  caught the error:

➜ ~ $ srss tui
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x841baa]

goroutine 1 [running]:
github.com/sheepla/srss/ui.humanizeTime(0x0)
	/home/runner/work/srss/srss/ui/util.go:121 +0x2a
github.com/sheepla/srss/ui.FindItem.func1(0xfd)
	/home/runner/work/srss/srss/ui/finder.go:18 +0x46
github.com/ktr0731/go-fuzzyfinder.(*finder).find.func1(0x210)
	/home/runner/go/pkg/mod/github.com/ktr0731/go-fuzzyfinder@v0.6.0/fuzzyfinder.go:604 +0x8a
github.com/ktr0731/go-fuzzyfinder.(*finder).find(0xc0001032b0, {0x89f500?, 0xc0008012c0}, 0xc0007fa9c0, {0xc0005f1bc0, 0x1, 0x7fb4389e6501?})
	/home/runner/go/pkg/mod/github.com/ktr0731/go-fuzzyfinder@v0.6.0/fuzzyfinder.go:646 +0x6bd
github.com/ktr0731/go-fuzzyfinder.(*finder).Find(0xc0007fa9c0?, {0x89f500?, 0xc0008012c0?}, 0xc0005f1b80?, {0xc0005f1bc0?, 0x20?, 0x8ddcc0?})
	/home/runner/go/pkg/mod/github.com/ktr0731/go-fuzzyfinder@v0.6.0/fuzzyfinder.go:726 +0x28
github.com/ktr0731/go-fuzzyfinder.Find({0x89f500, 0xc0008012c0}, 0x210?, {0xc0005f1bc0, 0x1, 0x1})
	/home/runner/go/pkg/mod/github.com/ktr0731/go-fuzzyfinder@v0.6.0/fuzzyfinder.go:722 +0x65
github.com/sheepla/srss/ui.FindItem({0xc0000be000, 0x210, 0x210})
	/home/runner/work/srss/srss/ui/finder.go:15 +0x119
main.initApp.func5(0xc000242240?)
	/home/runner/work/srss/srss/main.go:169 +0x110
github.com/urfave/cli/v2.(*Command).Run(0xc000242240, 0xc000244100)
	/home/runner/go/pkg/mod/github.com/urfave/cli/v2@v2.7.1/command.go:168 +0x631
github.com/urfave/cli/v2.(*App).RunContext(0xc000153380, {0xa12ec0?, 0xc0000280d8}, {0xc00001e060, 0x2, 0x2})
	/home/runner/go/pkg/mod/github.com/urfave/cli/v2@v2.7.1/app.go:320 +0xbc8
github.com/urfave/cli/v2.(*App).Run(...)
	/home/runner/go/pkg/mod/github.com/urfave/cli/v2@v2.7.1/app.go:226
main.main()
	/home/runner/work/srss/srss/main.go:50 +0x45

**Comment by sheepla** [comment]
Thank you for your reporting. Probably the `nil` check of the argument of the `humanizeTime()` function was leaked.

I tried to add testing code below:

```go
package ui

import (
	"testing"
)

func TestHumanizeTime(t *testing.T) {
	have := humanizeTime(nil)
	want := ""
	if have != want {
		t.Error("error!")
	}
}
```

And running the test caused a panic as well.

```
go test ./...
PASS | github.com/sheepla/srss
PASS | github.com/sheepla/srss/cache
SKIP | github.com/sheepla/srss/opml     [no test files]
FAIL | HumanizeTime (0.00s)
     | panic: runtime error: invalid memory address or nil pointer dereference [recovered]
     |  panic: runtime error: invalid memory address or nil pointer dereference
     | [signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x5f5cca]
     | goroutine 6 [running]:
     | testing.tRunner.func1.2({0x621960, 0x839880})
     |  /usr/lib/go/src/testing/testing.go:1396 +0x24e
     | testing.tRunner.func1()
     |  /usr/lib/go/src/testing/testing.go:1399 +0x39f
     | panic({0x621960, 0x839880})
     |  /usr/lib/go/src/runtime/panic.go:884 +0x212
     | github.com/sheepla/srss/ui.humanizeTime(0x0)
     |  /home/sheepla/ghq/github.com/sheepla/srss/ui/util.go:121 +0x2a
     | github.com/sheepla/srss/ui.TestHumanizeTime(0xc00015aea0)
     |  /home/sheepla/ghq/github.com/sheepla/srss/ui/util_test.go:8 +0x25
     | testing.tRunner(0xc00015aea0, 0x6989c0)
     |  /usr/lib/go/src/testing/testing.go:1446 +0x10b
     | created by testing.(*T).Run
     |  /usr/lib/go/src/testing/testing.go:1493 +0x35f
FAIL |  github.com/sheepla/srss/ui      0.006s
SKIP | github.com/sheepla/srss/urlentry [no test files]
```

**Comment by sheepla** [comment]
I fixed it at 6aa9650. I close this issue.
